### PR TITLE
Add --com to duplicate --pro

### PR DESCRIPTION
**Pull request by backspace**
This probably needs other changes…?

### CODE REVIEW

**Comment by MariadeAnton** [comment]
@backspace can you update your branch to avoid the merge conflict with master? I've updated our .travis.yml in https://github.com/travis-ci/travis.rb/pull/616 so this should work now 🎉

**Comment by backspace** [comment]
Done.

It’s my belief that

> We automatically publish a new development version after every successful build.

means every successful `master` build. But I figured out how to run it locally with the help of [this](https://gist.github.com/jendiamond/6128723):

```
bundle install
bundle exec bin/travis
```

Then when I run these commands, the output is the same!

```
bundle exec bin/travis token --pro
bundle exec bin/travis token --com
```

I think that’s sufficient to confirm that it works as desired. I’ll make another commit to update the documentation.

**Comment by backspace** [comment]
I added it to the documentation once with 49bc702. There are other places it could go too, as various commands have the `--pro` explanation attached. I’m not sure whether it’s useful to show it every time…?

**Comment by MariadeAnton** [comment]
Yay - it works 🎉 

```
bundle exec bin/travis whoami --com
You are MariadeAnton (María de Antón)
```

@backspace when you say various commands have the `--pro` information attached to them? Is this in our README file? Or built-in the CLI?

File: README.md [hunk 1]
```diff
@@ -1402,12 +1403,12 @@ $ travis status -qpx && cap deploy
 
 ### Pro and Enterprise
```
Comment (MariadeAnton) [inline README.md]: Let's replace `Pro and Enterprise` in this title with `Travis CI and Travis CI Enterprise` - what do you think?

File: README.md [hunk 2]
```diff
@@ -558,7 +559,7 @@ Repository commands have all the options [General API Commands](#general-api-com
 
 Additionally, you can specify the Repository to talk to by providing `--repo owner/name`. However, if you invoke the command inside a clone of the project, the client will figure out this option on its own. Note that it uses the tracked [git remote](http://www.kernel.org/pub/software/scm/git/docs/git-remote.html) for the current branch (and defaults to 'origin' if no tracking is set) to do so. You can use `--store-repo SLUG` once to override it permanently.
 
-It will also automatically pick [Travis Pro](https://travis-ci.com) if it is a private project. You can of course override this decission with `--pro`, `--org` or `--api-endpoint URL`
+It will also automatically pick [Travis Pro](https://travis-ci.com) if it is a private project. You can of course override this decission with `--com`, `--org` or `--api-endpoint URL`
```
Comment (MariadeAnton) [inline README.md]: I'd rather use the full URL here, thoughts? 

[travis-ci.com](https://travis-ci.com)

Comment (backspace) [reply README.md]: Do you mean replace it with this?

```
It will also automatically pick [travis-ci.com](https://travis-ci.com] if it…
```

**Comment by MariadeAnton** [comment]
Looks great! Merging this in for a pre-release, thanks a bundle @backspace!

**Pull request merged**